src/vendor/github.com/xelabs/go-mysqlstack/sqlparser/ast.go

```diff
@@ -191,8 +191,15 @@ type (
 		NewName TableName
 	}
 
+	// ExplainType is an enum for Explain-Type
+	ExplainType int8
```

Comment:
ExplainType int8

const (
	ExplainTypeEmpty ExplainType = iota
	ExplainTypeExtended
	ExplainTypePartitions
	ExplainTypeTree
	ExplainTypeJSON
	ExplainTypeTraditional
)

Reply:
The constants move to constants.go may be better and in the future we can keep this way.